### ISSUE TITLE
[Bug]: Flow service is throwing error when user sends msg through the Whatsapp channel 

### ISSUE THREAD

**Issue by vaibhavbhuva** [OpenNyAI/Jugalbandi-Manager#146]
### Describe the bug

Running Jugalbandi manager locally results in an error due to a recent update in `jb-manager-bot`. The new version doesn't expose the `FSMOutput` class, which is required by existing code in the FSM file.

### Steps to reproduce

Step 1: Try building and running any existing version of the Jugalbandi application.
Step 2: If the application runs successfully, send a "Hi" message through the WhatsApp channel.
Step 2: Observe the application logs for any relevant information or errors.

### Screenshots and logs

<img width="1676" alt="Screenshot 2024-07-16 at 2 58 41 PM" src="https://github.com/user-attachments/assets/0916d05d-bcb3-4b47-ab50-d4707ee5f278">

### Additional Information

Include the version number of `jb-manager-bot` alongside the import statement in `flow/src/__main__.py`. This ensures compatibility with the expected version.

In the published version of `jb-manager-bot`, ensure the `FSMOutput` class is still accessible for existing projects.

Here's the sample code:

flow/src/__main__.py
```python

async def install_or_update_bot(bot_config: BotConfig):
    bot_id = bot_config.bot_id
    bot_name = bot_config.bot_name
    fsm_code = bot_config.bot_fsm_code
    requirements_txt = (
        "openai\ncryptography\njb-manager-bot==<{PUBLISHED_VERSION_OF_JB_MANAGER_BOT}>\n" + bot_config.bot_requirements_txt.
    )
```

**Comment by kanak8278** [comment]
We have deployed a new version of JB-Manager-Bot and Quickstart(Change in the ChatBot Code)
[car_wash.py](https://github.com/OpenNyAI/Jugalbandi-Manager/blob/main/docs/tutorials/car_wash.py)
This should resolve the issue. @vaibhavbhuva

**Comment by shreypandey** [comment]
@kanak8278 Can you please update the bot installer to use version `0.2.0`. We will create a release after that to update the published images.

**Comment by vaibhavbhuva** [comment]
@shreypandey  I have fixed in this #154

**Comment by shreypandey** [comment]
Merged the fix in #154
